test/unittests/eof_test.cpp

```diff
@@ -34,33 +35,40 @@ TEST(eof, read_valid_eof1_header)
         uint16_t data_size;
         std::vector<uint16_t> code_sizes;
     };
-    std::string nops_255;
-    for (int i = 0; i < 255; ++i)
-        nops_255 += "5B";
-
-    std::string section_size_1_256;
-    for (int i = 0; i < 256; ++i)
-        section_size_1_256 += "0001";
 
+    std::string nops_255;
+    std::string section_size_256;
+    std::string code_sections_256;
     std::string section_types_256;
     for (int i = 0; i < 256; ++i)
+    {
+        section_size_256 += "0003";
```

Comment:
I found a more concise way of making these sequences, I think I'll make a separate PR with it.

Reply:
I did it in https://github.com/ethereum/evmone/pull/796, please rebase now